UI/ControlMappingScreen.cpp

```diff
@@ -428,22 +423,21 @@ bool KeyMappingNewMouseKeyDialog::axis(const AxisInput &axis) {
 	return true;
 }
 
+enum class StickHistoryViewType {
+	INPUT,
+	OUTPUT
+};
+
 class JoystickHistoryView : public UI::InertView {
 public:
-	JoystickHistoryView(int xAxis, int xDevice, int xDir, int yAxis, int yDevice, int yDir, UI::LayoutParams *layoutParams = nullptr)
-		: UI::InertView(layoutParams),
-			xAxis_(xAxis), xDir_(xDir),
-			yAxis_(yAxis), yDir_(yDir) {}
+	JoystickHistoryView(StickHistoryViewType type, std::string title, UI::LayoutParams *layoutParams = nullptr)
+		: UI::InertView(layoutParams), title_(title), type_(type) {}
 	void Draw(UIContext &dc) override;
-	std::string DescribeText() const override { return ""; }
+	std::string DescribeText() const override { return "Analog Stick View"; }
```

Comment:
This has a title_, so any description should include the title, and possibly values.  I was trying to use this semantically as an accessibility tree.  Using it as a developer name for the control is more the purpose of `DescribeLog()`.

I'd previously intentionally made it blank because the associated text was separate from this graphical representation, so this view had on separate and useful textual meaning.  Not sure if that's still true.

I realize it's not really exposed in a useful way to any accessibility tree at this point, and you probably wouldn't search it, but just wanted to explain the purpose of the method as I'd considered it.

-[Unknown]

Reply:
Oh right, this was just a driveby change and I didn't reflect on the real purpose. Thanks for clarifying!